fhir-server-test/src/test/java/com/ibm/fhir/server/test/operation/EverythingOperationTest.java

```diff
@@ -205,7 +205,7 @@ public void testPatientEverythingWithStartAndStop() {
         Bundle everythingBundle = response.readEntity(Bundle.class);
 
         // The number of companies was reduced as the scope was narrowed down to a decade
-        assertResponseBundle(everythingBundle, BundleType.SEARCHSET, 387);
```

Comment:
I was expecting some changes to the headers to direct against tenant1

Reply:
Got it.  Fixed.